operator/pkg/helmreconciler/render.go

```diff
@@ -386,6 +367,71 @@ func toChartManifestsMap(m name.ManifestMap) ChartManifestsMap {
 	return out
 }
 
+// getOwnerLabels returns a map of labels for the given component name, revision and owning CR resource name.
+func (h *HelmReconciler) getOwnerLabels(componentName string) (map[string]string, error) {
+	crName, err := h.getCRName()
+	if err != nil {
+		return nil, err
+	}
+	labels := make(map[string]string)
+	revision := ""
+	if h.iop != nil {
+		revision = h.iop.Spec.Revision
+	}
+
+	componentLabelValue := componentName
+	// Only pilot component uses revisions
+	if componentName == string(name.PilotComponentName) {
+		if revision == "" {
+			revision = "default"
+		}
+		labels[model.RevisionLabel] = revision
+		componentLabelValue += "-" + revision
```

Comment:
the `kubectl` model only uses this for `-l` for the prune -- not adding the label

Reply:
done